setup.py

```diff
@@ -2,128 +2,128 @@
+            includedir = os.path.abspath(os.path.join(scipoptdir, "include"))
         else:
-            print('Location SCIPOPTDIR does not contain an include directory; exiting because we cannot not find the include files.')
-            quit(1)
+            sys.exit(f"SCIPOPTDIR={scipoptdir} does not contain an include directory.")
 
     # determine library
-    if os.path.exists(os.path.join(scipoptdir, 'lib/shared/libscip.so')):
+    if os.path.exists(os.path.join(scipoptdir, "lib", "shared", "libscip.so")):
         # SCIP seems to be created with make
-        libdir = os.path.abspath(os.path.join(scipoptdir, 'lib/shared'))
-        libname = 'scip'
-        extra_compile_args.append('-DNO_CONFIG_HEADER')
+        libdir = os.path.abspath(os.path.join(scipoptdir, "lib", "shared"))
+        libname = "scip"
+        extra_compile_args.append("-DNO_CONFIG_HEADER")
         # the following is a temporary hack to make it compile with SCIP/make:
-        extra_compile_args.append('-DTPI_NONE') # this is just a guess, if other TPIs are used, please modify
+        extra_compile_args.append("-DTPI_NONE")  # if other TPIs are used, please modify
     else:
         # assume that SCIP is installed on the system
-        libdir = os.path.abspath(os.path.join(scipoptdir, 'lib'))
-        libname = 'scip'
-        if platform.system() in ['Windows']:
-            libname = 'libscip'
+        libdir = os.path.abspath(os.path.join(scipoptdir, "lib"))
+        libname = "libscip" if platform.system() in ["Windows"] else "scip"
 
-    print('Using include path <%s>.' % includedir)
-    print('Using SCIP library <%s> at <%s>.\n' % (libname,libdir))
+    print(f"Using include path {includedir}.")
+    print(f"Using SCIP library {libname} at {libdir}.\n")
 
 # set runtime libraries
-if platform.system() in ['Linux', 'Darwin']:
-    extra_link_args.append('-Wl,-rpath,'+libdir)
+if platform.system() in ["Linux", "Darwin"]:
+    extra_link_args.append(f"-Wl,-rpath,{libdir}")
 
 # enable debug mode if requested
 if "--debug" in sys.argv:
-    extra_compile_args.append('-UNDEBUG')
+    extra_compile_args.append("-UNDEBUG")
     sys.argv.remove("--debug")
 
 use_cython = True
 
-packagedir = os.path.join('src', 'pyscipopt')
+packagedir = os.path.join("src", "pyscipopt")
 
-with open(os.path.join(packagedir, '__init__.py'), 'r') as initfile:
-    version = re.search(r'^__version__\s*=\s*[\'"]([^\'"]*)[\'"]',
-                        initfile.read(), re.MULTILINE).group(1)
+with open(os.path.join(packagedir, "__init__.py"), "r") as initfile:
+    version = re.search(
+        r'^__version__\s*=\s*[\'"]([^\'"]*)[\'"]', initfile.read(), re.MULTILINE
+    ).group(1)
 
 try:
     from Cython.Build import cythonize
 except ImportError as err:
     # if cython is not found _and_ src/pyscipopt/scip.c does not exist then we cannot do anything.
-    if not os.path.exists(os.path.join(packagedir, 'scip.c')):
-        print('Cython is required')
-        quit(1)
```

Comment:
did you take this line out on purpose?

Reply:
yeah, `sys.exit` is preferred over `quit` according to "the internet" and it has the added benefit of printing a message. So we save one line and have slightly cleaner code.

Reply:
okay, i cannot find the sys.exit though

Reply:
![image](https://user-images.githubusercontent.com/1506995/152697066-e467c1d0-e3e9-4f39-bf3c-40718552c5f5.png)

I just replaced single quotes with double quotes and added sys.exit.

Reply:
ah yes, i was too close to the blackboard, all is good!